.github/CODEOWNERS

```diff
@@ -56,11 +56,11 @@
 
 # Ray data.
 /python/ray/data/ @ericl @scv119 @clarkzinzow @jjyao
-/doc/source/data/ @ericl @scv119 @clarkzinzow @jjyao
+/doc/source/data/ @ericl @scv119 @clarkzinzow @jjyao @maxpumperla
```

Comment:
thanks @krfricke ! We should maybe think about following this up with a PR that deletes these two specific lines and adds 3-4 maintainers of the _whole_ `/doc` package.

We've had a couple of unnecessary regressions / deviations that I'd like to put a safeguard on.

Reply:
+1 that would be preferable to me

Reply:
Sounds good - let's do this in a follow up? It would be great if you could suggest these people @maxpumperla